change/@fluentui-react-spinbutton-5b913def-1963-4526-9944-465231bd61f8.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat: update spinbutton to stable",
```

Ditto the comment above:

```suggestion
  "comment": "chore: update spinbutton to stable",
```